workspace.json

```diff
@@ -10,6 +10,10 @@
     "dh-globalization-data-access-localization": "libs/dh/globalization/data-access-localization",
     "dh-globalization-domain": "libs/dh/globalization/domain",
     "dh-globalization-feature-language-picker": "libs/dh/globalization/feature-language-picker",
+    "dh-metering-point-feature-child": "libs/dh/metering-point/feature-child",
+    "dh-metering-point-feature-details": "libs/dh/metering-point/feature-details",
```

Comment:
question: What do you think about renaming the library `dh-metering-point-feature-details` to `dh-metering-point-feature-overview` because the page gives an overview of a metering point. Details can be used for more specific things on the page.

In the same manner, `dh-metering-point-feature-child` can become `dh-metering-point-feature-child-overview`.

Reply:
Should be done now